lib/ide/json-rpc/src/transport.rs

```diff
@@ -26,5 +26,7 @@ pub enum TransportEvent {
     /// A text message with has been received.
     TextMessage(String),
     /// A socket has been closed by the peer.
+    Opened,
```

Comment:
Where does it come from? Anyway, the docs should be fixed (it's " A socket has been **closed** by the peer." twice)

Reply:
idk, but this was fixed